apps/api/src/config/base.ts

```diff
@@ -104,8 +104,8 @@ export class BaseConfig implements IBaseConfig {
       // can't be 0 because LRUCache doesn't allow that
       1 / 1024 / 1024
     )
-    this.ENABLE_CUSTOM_OAI_KEY = this.getBooleanVar(
-      'ENABLE_CUSTOM_OAI_KEY',
+    this.DISABLE_CUSTOM_OAI_KEY = this.getBooleanVar(
+      'DISABLE_CUSTOM_OAI_KEY',
       true
```

Comment:
This second argument is the one we use when variable is not present, I think we want `false` right?